sagemaker-remote-function/pytorch_mnist_sample_script/config.yaml

```diff
@@ -5,6 +5,6 @@ SageMaker:
       RemoteFunction:
         # role arn is not required if in SageMaker Notebook instance or SageMaker Studio
         # Uncomment the following line and replace with the right execution role if in a local IDE
-        RoleArn: arn:aws:iam::471094633457:role/SageMakerExecutionRole
+        # RoleArn: <replace the role arn here>
         InstanceType: ml.m5.xlarge
         Dependencies: ./requirements.txt
```

Comment:
How about this action item?
> Use workdir_config or the new for PyTorch example

Reply:
Ah good catch. But shouldn’t this be added to the new notebooks given it’s apart of the new lightsaber functionality ? + Do we have any current workdir_config examples that we can modify ?

Reply:
It is for both @remote and @step.

Reply:
Yea. So I assume if we add this example here as apart of this notebook, do we want to include something similar in our new notebooks for using `CustomFileFilter` with @steps